### PR TITLE
Insert more information so that other users can read it.

### PR DESCRIPTION
**Pull request by huavanthong**
Hi Thomas,

I have updated the additional information as per Holger's comments https://github.com/test-fullautomation/RobotFramework_AIO/issues/180#issuecomment-1821096018
If you have any useful information that should be added for other users, please let me know.

Best regards,
Thong Hua

### CODE REVIEW

File: build [hunk 1]
```diff
@@ -254,8 +252,33 @@ function parse_config () {
...
+	add_line_to_file "********************"
+	add_line_to_file "RobotFramework AIO Tool full version:"
+	add_line_to_file "    https://htmlpreview.github.io/?https://github.com/test-fullautomation/robotframework-documentation/blob/develop/RobotFrameworkAIO/Components.html"
+	add_line_to_file ""
+	add_line_to_file "RobotFramework AIO project:"
+	add_line_to_file "    https://github.com/test-fullautomation/RobotFramework_AIO"
+	add_line_to_file ""
+	add_line_to_file "Author:"
+	add_line_to_file "********************"
+	add_line_to_file "Thomas Pollerspöck <Pollerspoeck@de.bosch.com>"
```
Comment (test-fullautomation) [inline build]: Hi Thong,
correct is:
thomas.pollerspoeck@de.bosch.com
Thank you,
 Thomas

File: build [hunk 2]
```diff
@@ -304,7 +327,7 @@ function build_debian()
 	cp ./config/build/dpkg_build/postinst.sh ./output_lx/${PACK_NAME}/usr/local/bin/initRobotFrameworkAIO.sh
 
 	# Update version
-	cp ./version.txt ./output_lx/${PACK_NAME}/opt/rfwaio/
+	cp ./Readme.txt ./output_lx/${PACK_NAME}/opt/rfwaio/
```
Comment (test-fullautomation) [inline build]: Hi Thong,
please let the name "version.txt".
I want to highlight that the version can be found here.
Readme can be a lot of things...
Thank you,
 Thomas

File: scripts/RobotFrameworkSetup.iss [hunk 3]
```diff
@@ -89,7 +89,7 @@ Source: .\PowerShell\update_vsdata.ps1; DestDir: "{tmp}"; Flags: ignoreversion;
 ;;;
 
 ; Update version
-Source: ..\version.txt; DestDir: {app}; Flags: ignoreversion recursesubdirs createallsubdirs; Permissions: everyone-full;
+Source: ..\Readme.txt; DestDir: {app}; Flags: ignoreversion recursesubdirs createallsubdirs; Permissions: everyone-full;
```
Comment (test-fullautomation) [inline scripts/RobotFrameworkSetup.iss]: same like before.

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Thong,
looks good to me!
Please check my findings.
Thank you,
 Thomas

**Comment by huavanthong** [comment]
Hi Thomas, 

I have updated it following your comments. 
Please help me merge this request.

Thank you,
Thong Hua

**Comment by test-fullautomation** [comment]
Hi @huavanthong ,

what about renaming the file to version.txt ?

Thank you,
 Thomas

**Comment by huavanthong** [comment]
Hi Thomas,

I have updated it at commit https://github.com/test-fullautomation/RobotFramework_AIO/pull/188/commits/a41dd9eb980a5f8a1deda85b2001a305932ebe9c
Could you help me review it?
I’m sorry because I forgot to fix it at your comment directly.

Thank you, 
Thong Hua

**Pull request merged**